README.md

```diff
@@ -20,16 +20,24 @@ ProfileFolio simplifies professional profile management by generating customized
 <details>
 <summary><strong>Install Manually</strong></summary>
 
-- Open Mongodb atlas in localhost:27017
-  Create `profileFolio` db and
+> [!IMPORTANT]  
+> These installation steps assume you already have Go, MongoDB and NodeJS installed on your machine.
+> * Node JS - https://nodejs.org/en
+> * GO      - https://go.dev/doc/install
+> * MongoDB - https://www.mongodb.com/try/download/shell
+
 
-1.  Add this sample data in collection `users`
+ Open Mongodb atlas in localhost:27017
+  Create `profileFolio` db and
```

Comment:
Why do we need this? When we have a bat file?

Reply:
I feel we can still leave it, just in case for some reason if the batch file does not work people would still be able to refer these steps.